examples/immutable/components/user-form.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
```

Comment:
I just want to point out that I left the validation stuff out of this example for now until I can get the Errors component updated to being compatible with Immutable.